skills/common/process/merge/runtime/git-operations.sh

```
@@ -0,0 +1,331 @@
+
+    echo "Removing Worktree: $_worktree"
+
+    if [ ! -d "$_worktree" ]; then
+        echo "WARNING: Worktree does not exist: $_worktree" >&2
+    else
+        if [ "$_force" = "true" ]; then
+            git worktree remove --force "$_worktree" 2>/dev/null
+        else
+            git worktree remove "$_worktree" 2>/dev/null
+        fi
+        if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)